py-polars/polars/lazyframe/frame.py

```diff
@@ -5172,20 +5172,27 @@ def map_batches(
         --------
         >>> lf = pl.LazyFrame(
         ...     {
-        ...         "a": [1, 2],
-        ...         "b": [3, 4],
+        ...         "a": range(-100_000, 0),
+        ...         "b": range(0, 100_000),
```

Comment:
Shall we use `int_range` here?

```suggestion
        ...         "a": pl.int_range(-100_000, 0),
        ...         "b": pl.int_range(0, 100_000),
```

Reply:
Sure, can do - we do actually recognise/specialise python `range` internally though (for a huge speedup, of course :), so there's no performance difference, but it's a good idea to draw attention to the native variant 👍

Reply:
> so there's no performance difference

Ah, I didn't know that. If I knew that, I would've left this. But either is fine I guess.

Reply:
> Ah, I didn't know that. If I knew that, I would've left this. But either is fine I guess.

I only know about it as it was one of mine...🤣 
https://github.com/pola-rs/polars/pull/5397